app/pages/sections/commencement-date/controller.js

```diff
@@ -1,3 +1,10 @@
 import { getContext } from './contextCreator';
+import { getDateErrors } from './getDateErrors';
 
 export const getCommencementDateContext = async parmas => getContext(parmas);
+
+export const putCommencementDate = async ({ data }) => {
+  const errors = [getDateErrors(data)];
+  if (errors[0]) return { success: false, errors };
```

Comment:
`errors.length > 0` perhaps?

Reply:
nope - counts the null as an entry in the array